pkg/log/log.go

```diff
@@ -0,0 +1,46 @@
+package log
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+type LogLevel string
+
+const (
+	LogLevelInfo  LogLevel = "info"
+	LogLevelWarn  LogLevel = "warn"
+	LogLevelError LogLevel = "error"
+	LogLevelFatal LogLevel = "fatal"
+)
+
+// Log prints tagged log messages. It is a stub for future reporting of
+// such messages to the nebula service
+func Log(writer io.Writer, level LogLevel, log string) {
```

Comment:
I think I would make this package-private for now so we can really work out what features we want to support here (e.g., where to add tagging, etc.)